examples/sdk-examples/package.json

```diff
@@ -64,7 +64,6 @@
         "eslint-plugin-sonarjs": "^0.16.0",
         "eslint-plugin-tsdoc": "^0.2.14",
         "fork-ts-checker-webpack-plugin": "^9.0.2",
-        "heroku": "8.7.1",
```

Comment:
we can perhaps also remove https://github.com/gooddata/gooddata-ui-sdk/blob/master/common/scripts/ci/docker_examples.sh and https://github.com/gooddata/gooddata-ui-sdk/blob/master/common/scripts/ci/run_release_examples.sh

Reply:
done